tripy/tests/frontend/test_utils.py

```diff
@@ -171,29 +179,35 @@ def test_convert_shape_unsqueeze_tensors(self):
         assert isinstance(s, tp.Shape)
         assert cp.from_dlpack(s).get().tolist() == [1, 2]
 
+    def test_convert_only_named_argument_to_shape(self):
```

Comment:
nit: maybe "specified" instead of "named" so as not to confuse it with named in the sense of named vs. positional. 
```suggestion
    def test_convert_only_specified_argument_to_shape(self):
```